worker-archetype/src/main/resources/archetype-resources/__rootArtifactId__-container/pom.xml

```diff
@@ -63,13 +63,7 @@
         <dependency>
             <groupId>com.github.workerframework</groupId>
             <artifactId>standard-worker-container</artifactId>
-            <type>pom</type>
-            <exclusions>
-                <exclusion>
-                    <groupId>com.github.workerframework</groupId>
-                    <artifactId>worker-store-cs</artifactId>
-                </exclusion>
-            </exclusions>
+            <type>pom</type>            
```

Comment:
Yea, some whitespace was added to the end. I will remove it in the next feedback commit